lib/src/clojure_lsp/queries.clj

```diff
@@ -24,24 +23,22 @@
     nil
     coll))
 
-(defn filter-project-analysis-xf [db]
```

Comment:
Could be a `def`. Same for `filter-external-analysis-xf`.

Reply:
Makes sense!